### PR TITLE
feat(database): add script to build local database

### PR DESCRIPTION
**Pull request by quannhg**
Dựa trên file script khi trưa anh Vinh viết để viết một file typescript xây dựng User Account Database

### CODE REVIEW

**Review by ngyngcphu — COMMENTED** [review]
no problem

**Comment by quannhg** [comment]
This's branch is having bug, don't merge

**Comment by quannhg** [comment]
The bug is fixed, can merge now

**Review by ngyngcphu — COMMENTED** [review]
ok

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    db:
+      image: mysql:latest
+      container_name: mysql
+      restart: always
+      ports:
+        - 3306:3306
+      volumes:
+        - tickflowdb:/var/lib/mysql
+      environment:
+        MYSQL_ROOT_PASSWORD: ${ACCOUNT_DB_ROOT_PASSWORD}
```
Comment (ngyngcphu) [inline docker-compose.yml:12]: fix successly

**Review by ngyngcphu — APPROVED** [review]

**Pull request merged**